resources/ts/Components/UitIdWidget.tsx

```diff
@@ -63,6 +63,12 @@ export const UitIdWidget = ({
     [component]
   );
 
+  const currentPageToVariables = {
+    "integrations/detail": {
+      id: props.integration?.id,
+    },
+  };
+
```

Comment:
```suggestion
  const currentPageToVariables: { [key: string]: unknown } = {
    [key: string]: unknown;
    
    "integrations/detail": {
      id: props.integration?.id,
    },
  };

```

This will satisfy the eslint error